src/bedriftsmeny/Virksomhetsvelger/byggSokeresultat.ts

```diff
@@ -1,53 +1,77 @@
 import fuzzysort from 'fuzzysort';
```

Comment:
i søkefeltversjonen blir treffet for juridisk enhet åpen hvis den inneholder underenheten som allerede er valgt, slik du sa. men kan hende det er best brukeopplevelse?